src/main/java/fr/esgi/al5_2/Tayarim/dto/proprietaire/AdministrateurDTO.java

```diff
@@ -0,0 +1,29 @@
+package fr.esgi.al5_2.Tayarim.dto.proprietaire;
+
+import fr.esgi.al5_2.Tayarim.dto.logement.LogementDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+import java.time.LocalDateTime;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.time.LocalDateTime' import. Should be before 'lombok.NonNull'.